### ISSUE TITLE
Typeerror: GeoJSON is not a constructor

### ISSUE THREAD

**Issue by rahulganguly** [RideAmigosCorp/mongoose-geojson-schema#19]
I was trying out the sample code on version 2.1.2 and I keep getting the following error
Typeerror: GeoJSON is not a constructor

 var test = new GeoJSON({
        any: {
            type: "Point",
            point: [-113.806458, 44.847784]
        },
        point: {
            type: "Point",
            coordinates: [12.123456, 13.134578]
        }
    });

**Comment by muhammadfaizan** [comment]
yeah that's an issue, you can use your model name, just like any other model you'd create.
